ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3764313&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3764313&tab=artifacts#/) on master @ [010cf883d8b3ed2721b726df1cf8060777955d41](https://github.com/cockroachdb/cockroach/commits/010cf883d8b3ed2721b726df1cf8060777955d41):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1948538961
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1948538961
--- FAIL: TestChangefeedBackfillCheckpoint (10.39s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (2.97s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3765044&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3765044&tab=artifacts#/) on master @ [e131a825651c429e322fbd97ff0ed7b43ecccc8a](https://github.com/cockroachdb/cockroach/commits/e131a825651c429e322fbd97ff0ed7b43ecccc8a):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint3093958790
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint3093958790
--- FAIL: TestChangefeedBackfillCheckpoint (7.17s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (1.95s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3765452&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3765452&tab=artifacts#/) on master @ [d5ffc1429c4dbcd462209e37666135fcc9405cdf](https://github.com/cockroachdb/cockroach/commits/d5ffc1429c4dbcd462209e37666135fcc9405cdf):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1001225042
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1001225042
--- FAIL: TestChangefeedBackfillCheckpoint (12.61s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB (4.11s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3766433&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3766433&tab=artifacts#/) on master @ [d5ffc1429c4dbcd462209e37666135fcc9405cdf](https://github.com/cockroachdb/cockroach/commits/d5ffc1429c4dbcd462209e37666135fcc9405cdf):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint871952149
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint871952149
--- FAIL: TestChangefeedBackfillCheckpoint (8.14s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (2.28s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3766893&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3766893&tab=artifacts#/) on master @ [8a52599fd86836644936ddd4d138db88f6a647bf](https://github.com/cockroachdb/cockroach/commits/8a52599fd86836644936ddd4d138db88f6a647bf):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint60760933
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint60760933
--- FAIL: TestChangefeedBackfillCheckpoint (9.48s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (2.85s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767007&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767007&tab=artifacts#/) on master @ [429fee08d8b2bbb63ee245858b8d3e9180a57145](https://github.com/cockroachdb/cockroach/commits/429fee08d8b2bbb63ee245858b8d3e9180a57145):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint3118260894
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint3118260894
--- FAIL: TestChangefeedBackfillCheckpoint (9.85s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB (2.42s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767263&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767263&tab=artifacts#/) on master @ [429fee08d8b2bbb63ee245858b8d3e9180a57145](https://github.com/cockroachdb/cockroach/commits/429fee08d8b2bbb63ee245858b8d3e9180a57145):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint4073040765
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint4073040765
--- FAIL: TestChangefeedBackfillCheckpoint (8.18s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (2.20s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767340&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767340&tab=artifacts#/) on master @ [8ef90ca92676389da2c955054ab5d996bb9ae6ff](https://github.com/cockroachdb/cockroach/commits/8ef90ca92676389da2c955054ab5d996bb9ae6ff):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1554540379
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4617: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1554540379
--- FAIL: TestChangefeedBackfillCheckpoint (11.10s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4544: 
        	Error Trace:	changefeed_test.go:4544
        	            				changefeed_test.go:4608
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (3.95s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767365&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767365&tab=artifacts#/) on master @ [8a52599fd86836644936ddd4d138db88f6a647bf](https://github.com/cockroachdb/cockroach/commits/8a52599fd86836644936ddd4d138db88f6a647bf):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint449718214
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4612: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint449718214
--- FAIL: TestChangefeedBackfillCheckpoint (8.83s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4539: 
        	Error Trace:	changefeed_test.go:4539
        	            				changefeed_test.go:4603
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (3.23s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767698&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3767698&tab=artifacts#/) on master @ [8ef90ca92676389da2c955054ab5d996bb9ae6ff](https://github.com/cockroachdb/cockroach/commits/8ef90ca92676389da2c955054ab5d996bb9ae6ff):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1106199728
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4617: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1106199728
--- FAIL: TestChangefeedBackfillCheckpoint (13.77s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4544: 
        	Error Trace:	changefeed_test.go:4544
        	            				changefeed_test.go:4608
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (3.89s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=deadlock

- GOFLAGS=-parallel=4
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3770378&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3770378&tab=artifacts#/) on master @ [6abeda76d5ed629cf52aeac2e943aa55cf7212fc](https://github.com/cockroachdb/cockroach/commits/6abeda76d5ed629cf52aeac2e943aa55cf7212fc):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint528407439
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4617: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint528407439
--- FAIL: TestChangefeedBackfillCheckpoint (18.58s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
    changefeed_test.go:4544: 
        	Error Trace:	changefeed_test.go:4544
        	            				changefeed_test.go:4608
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB (5.75s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=deadlock

- GOFLAGS=-parallel=4
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3770377&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3770377&tab=artifacts#/) on master @ [6abeda76d5ed629cf52aeac2e943aa55cf7212fc](https://github.com/cockroachdb/cockroach/commits/6abeda76d5ed629cf52aeac2e943aa55cf7212fc):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint3482295
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4617: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint3482295
--- FAIL: TestChangefeedBackfillCheckpoint (3.90s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
    changefeed_test.go:4544: 
        	Error Trace:	changefeed_test.go:4544
        	            				changefeed_test.go:4608
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=100_B
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=100_B (1.20s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3774384&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3774384&tab=artifacts#/) on master @ [cfb0bf33148744ec06d5d91a1927ca3115101782](https://github.com/cockroachdb/cockroach/commits/cfb0bf33148744ec06d5d91a1927ca3115101782):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1060756722
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4617: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint1060756722
--- FAIL: TestChangefeedBackfillCheckpoint (3.50s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
    changefeed_test.go:4544: 
        	Error Trace:	changefeed_test.go:4544
        	            				changefeed_test.go:4608
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB (1.15s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3774385&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3774385&tab=artifacts#/) on master @ [cfb0bf33148744ec06d5d91a1927ca3115101782](https://github.com/cockroachdb/cockroach/commits/cfb0bf33148744ec06d5d91a1927ca3115101782):


```
=== RUN   TestChangefeedBackfillCheckpoint
    test_log_scope.go:79: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint33033003
    test_log_scope.go:80: use -show-logs to present logs inline
=== CONT  TestChangefeedBackfillCheckpoint
    changefeed_test.go:4617: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestChangefeedBackfillCheckpoint33033003
--- FAIL: TestChangefeedBackfillCheckpoint (18.73s)
=== RUN   TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
    changefeed_test.go:4544: 
        	Error Trace:	changefeed_test.go:4544
        	            				changefeed_test.go:4608
        	            				helpers_test.go:545
        	Error:      	Should be true
        	Test:       	TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB
        	Messages:   	err=sql: database is closed
    --- FAIL: TestChangefeedBackfillCheckpoint/enterprise-limit=105_MB (5.79s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=deadlock

- GOFLAGS=-parallel=4
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73005 ccl/changefeedccl: TestChangefeedBackfillCheckpoint failed [C-test-failure O-robot T-cdc branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -4541,7 +4541,7 @@ func TestChangefeedBackfillCheckpoint(t *testing.T) {
 		defer func() {
 			closeFeed(t, foo)
 			if err := g.Wait(); err != nil {
-				require.Truef(t, jobs.HasErrJobCanceled(err), "err=%v", err)
+				require.NotRegexp(t, "unexpected epoch resolved event", err)
 			}
 		}()
 

